docs/en/getters.md

```diff
@@ -61,6 +61,27 @@ computed: {
 }
 ```
 
+You can also return functions in getters:
+```js
+getters: {
+  // ...
+  getTodoById: (state, getters) => (id) => {
+    return getters.todos.find(todo => todo.id === id)
+  }
+  // function syntax
+  getTodoById: function(state, getters){
```

Comment:
Honestly, I didn't I know you could do this with getters until I saw this commit. I'm sure many like myself will also benefit from this example.